### DISCUSSION TITLE
How to force refresh the Tutanota client desktop app?

### DISCUSSION THREAD

**Discussion by Himujjal** [tutao/tutanota#4548] [Bugs]
I use an updated app on MacOS M1

I am not able to update the emails though. The auto-update feature doesn't seem to work most of the time. For example, if I delete. an email, it still shows up in the inbox and I have to manually close and open the client.

Its a problem with the auto-refresher. So, either it has to be fixed, or a manual refresh button has to be provided.

I am a premium customer and I am frustrated by this issue.

**Comment by charlag** [comment]
Hi
There's no manual refresh because we rely on the real-time updates, if they are not working then something is wrong.
Does it happen all the time for you? Do you see a connection indicator in the top bar and what does it say?
Normally it should be just a Websocket connection unless your network somehow blocks it.

**Reply by Himujjal** [reply]
The connection indicator shows `online` but the emails are not getting refreshed automatically.

And this recently happened after a few updates.

**Comment by Himujjal** [comment]
Any updates on this? This is not resolved still

**Reply by charlag** [reply]
Your websocket is not actually connected or not working. This is not normal behavior.
